README.md

```diff
@@ -1 +1,17 @@
 # buildpack-static-require
+
+This buildpack complements the
+[`buildpack-static-confgen`](https://github.com/ninech/buildpack-static-confgen)
+buildpack, see the `README.md` on why this can't be in a single buildpack.
```

Comment:
I assume you mean the `README.md` in the other repo, because I am just reading a `README.md` file?

Reply:
yep, is it not clear enough with the link just before that?

Reply:
Yeah it is fine.